core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -375,8 +376,8 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
           OperandTypes.STRING_INTEGER_OPTIONAL_INTEGER,
           SqlFunctionCategory.STRING);
 
-  /** The "ENDS_WITH(value1, value2)" function (BigQuery, PostgreSQL). */
-  @LibraryOperator(libraries = {BIG_QUERY, POSTGRESQL})
+  /** The "ENDS_WITH(value1, value2)" function (BigQuery). */
```

Comment:
Please amend your commit message with a list of functions added + removed from which libraries. For instance, it's important that you note that after this change the Postgres library no longer contains ENDS_WITH.

There should also be a line "Redshift library is as Postgres, but adds GREATEST, LEAST, ... and removes ARRAY_AGG, ...".

Reply:
I have updated the commit message to reflect what is added and removed from function libraries. Also mentioned that the Amazon Redshift library extends the PostgreSQL function library.